alldogs.html

```diff
@@ -0,0 +1,164 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <title>Adoptadog - Seattle's premiere dog adoption agency</title>
+    <link rel="stylesheet" href="css/main.css">
+  </head>
+  <body>
+    <header id='page-header'>
+      <a href='#start' class='skiplink'>Skip to content</a>
+      <div id='identity'>
+        <img class='logo' src="images/adoptadog-logo.svg" alt="Adopt a Dog logo">
+        <p class='tagline'>Seattle's premiere dog adoption agency</p>
+      </div>
+      <div id='cart-bug'>
+          <img class='cart-icon' src="images/cart-icon.svg" alt="Shopping cart icon">
+          <span id='cart-total'>$1234.56</span>
+          <a href="#" class='action-button'>Checkout</a>
+      </div>
+      <nav>
+          <ul>
+            <li><a href="index.html">Home</a></li>
+            <li><a href="#">Dogs</a></li>
+            <li><a href="blog.html">Blog</a></li>
+          </ul>
+      </nav>
+    </header>
+    <main>
+      <section>
+        <header>
+          <h1 id='start'>Here are all of our Dogs!</h1>
+        </header>
+        <article class="">
```

Comment:
Be careful to use article tags on things that are actual articles. I'm not sure a wrapping container is the best use of article

Reply:
Ok. I don't need a wrapper, I just included it anticipating that pages will change over time. If in the future there were, say, two FAQ blurbs (which I would likely use article for) added to the end of the page, would we then want the dog list to be an article, too?

Reply:
I'd argue that a list of dog tiles isn't semantically an article, but just a wrapping container. I like to use articles on things that are actually articles. A wrapping container is good though, but I just like to keep article elements to articles